src/python/pants/option/options_bootstrapper_test.py

```diff
@@ -3,103 +3,123 @@
 
```

Comment:
This file has some refactoring. Pardon that it resulted in a bigger diff than ideal.